i18n/en.json

```diff
@@ -2902,14 +2902,14 @@
   "post_info.message.visible": "(Only visible to you)",
   "post_info.message.visible.compact": " (Only visible to you)",
   "post_info.permalink": "Permalink",
-  "post_info.pin": "Pin to channel",
+  "post_info.pin": "Pin to Channel",
   "post_info.pinned": "Pinned",
   "post_info.reply": "Reply",
   "post_info.submenu.icon": "submenu icon",
   "post_info.submenu.mobile": "mobile submenu",
   "post_info.system": "System",
   "post_info.tooltip.add_reactions": "Add Reaction",
-  "post_info.unpin": "Un-pin from channel",
+  "post_info.unpin": "Un-pin from Channel",
```

Comment:
Question: Is "un-pin" more common than "unpin"?

Reply:
I wasn't sure about this, so left it in. Merriam-Webster says 'unpin' and I myself use hyphens sparingly for prefixes, so would've preferred 'unpin', but I still left it in. Should I change the instances to 'unpin'?

Reply:
I'm partial to unpin, so I support changing it to "unpin".

Reply:
I've changed it to 'Unpin'. Resolving this conversation.